eZ/Publish/API/Repository/Values/Content/Content.php

```diff
@@ -28,6 +28,18 @@
     abstract public function getVersionInfo();
 
     /**
+     * Shorthand method for getVersionInfo()->getName().
+     *
+     * @see \eZ\Publish\API\Repository\Values\Content\VersionInfo::getName()
+     *
+     * @param string|null $languageCode
+     *
+     * @return string|null The name for a given language, or null if $languageCode is not set
+     *         or does not exist.
+     */
+    abstract public function getName($languageCode = null);
```

Comment:
Could do that

However as before when it comes to API BC promise, in #2002 didn't really consider implementations of API our BC promise, but rather consumption. So there are several breaks in the previous PR already to be able to archive this feature:
1. lots of service methods has new `$prioritizedLanguages` argument for languages where it was missing
2. `$language` property on `getName` and `getDescription` is now made optional on all value objects
3. `ContentTypeGroup` now has the name and description methods on API, they where not there from before, only in Core